### ISSUE TITLE
Trouble referencing Sync and OsuRTDataProvider for build

### ISSUE THREAD

**Issue by kk1995** [KedamaOvO/RealTimePPDisplayer#4]
I want to fork it and try some things out but I am having trouble creating RealTimePPDisplayer DLL because it cannot reference Sync and OsuRTDataProvider.

I solved the problem with OsuRTDataProvider by editing the .csproj file to reference the DLL.

```
<Reference Include="OsuRTDataProvider">
	<HintPath>D:\GitHub\OsuRTDataProvider\Plugins\OsuRTDataProviderRelease.dll</HintPath>
</Reference>
```

However, I'm not sure what to do with Sync, especially because Sync is an executable and not a library. I tried doing the same thing with Sync on the .csproj file but it did not work.

```
<ProjectReference Include="D:\GitHub\Sync-2.14.0\Sync\Sync.csproj">
      <Project>{fbd514c2-2830-479e-b050-d1c383028456}</Project>
      <Name>Sync</Name>
```

 Sorry this is my first time fiddling around with C# so if you could help me out that would be great.

TLDR: I want to make a DLL out of modified RealTimePPDisplayer project and try it out on Sync.exe, but I cannot make the DLL.

**Comment by KedamaOvO** [comment]
Sync: https://github.com/Deliay/Sync

**Comment by kk1995** [comment]
Thank you!

For anyone who reads this, changing part of .csproj to 

```
    <ProjectReference Include="D:\GitHub\Sync-master\Sync-master\Sync\Sync.csproj">
      <Project>{fbd514c2-2830-479e-b050-d1c383028456}</Project>
      <Name>Sync</Name>
    </ProjectReference>
```
made it work. Just link to where you have Sync.csproj

**Comment by KedamaOvO** [comment]
Add reference from Sync.exe.  

step 1:![](https://i.imgur.com/npblgBB.png)
setp 2:![](https://i.imgur.com/hPI3spV.png)
setp 3:![](https://i.imgur.com/H2tLqKPg.png)
step 4:![](https://i.imgur.com/vt02QClg.png)

**Comment by tsitra0** [comment]
Thank you for the much simpler solution!

**Comment by tsitra0** [comment]
Oh and I am same as kk1995 but I prefer this account for non work stuff.
